requirements-dev.txt

```
@@ -15,3 +15,4 @@ sphinx-rtd-theme>=0.4.3
 ufmt==1.3
 usort==1.0.0rc1
 setuptools-rust>=0.12.1
+slotscheck>=0.7.1,<0.8
```

I'm fine with an unbounded dependency here; also it's OK to only install it on python > 3.7
```suggestion
slotscheck>=0.7.1; python_version >= 3.7
```

---

Oh I see you released 0.7.1 for 3.6 compat. In that case, just
```suggestion
slotscheck>=0.7.1
```